tests/e2e/backup_restore_suite_test.go

```diff
@@ -101,9 +101,12 @@ var _ = Describe("AWS backup restore tests", func() {
 				Eventually(AreResticPodsRunning(namespace), timeoutMultiplier*time.Minute*3, time.Second*5).Should(BeTrue())
 			}
 			if brCase.BackupRestoreType == CSI {
-				log.Printf("Creating VolumeSnapshotClass for CSI backuprestore of %s", brCase.Name)
-				err = InstallApplication(dpaCR.Client, "./sample-applications/gp2-csi/volumeSnapshotClass.yaml")
-				Expect(err).ToNot(HaveOccurred())
+				if provider == "aws" || provider == "ibmcloud" {
+					log.Printf("Creating VolumeSnapshot for CSI backuprestore of %s", brCase.Name)
```

Comment:
Please keep the log name as VolumeSnapshotClass

Reply:
```suggestion
					log.Printf("Creating VolumeSnapshotClass for CSI backuprestore of %s", brCase.Name)
```

Reply:
No pb, I didn't notice the change when I rebased on master.